recipe/meta.yaml

```diff
@@ -107,33 +92,34 @@ requirements:
     - libspatialindex
     - libspatialite
     # For PostgreSQL support.
-    - postgresql
+    - postgresql >=14,<15
     - psycopg2
+    # Point Clouds
+    - pdal
+    - laz-perf
   run:
     - python
     - icu
     - libzip
     - libtasn1  # [osx]
-    - qt
+    - qt-main
     - pyqt
     - qtwebkit
     - qscintilla2
     - qca
     - qjson
     - qwt
-    - qwtpolar
     - qtkeychain
-    - qtlocation
-    - qtserialport  # [unix]
-    - gdal
+    - {{ pin_compatible('libgdal', max_pin='x.x') }}
+    - {{ pin_compatible('gdal', max_pin='x.x') }}
```

Comment:
I guess it can go back to just `libgdal` and `gdal` ?

BTW If these were missing from the `host` section earlier this would explain why we were getting gdal 3.6...